Modules/Module_Processing/Module_Population/xASL_im_CreateAnalysisMask.m

```diff
@@ -49,10 +49,11 @@
 bSkipStandard = 0;
 
 if x.dataset.nSubjectsSessions<16
-    fprintf('%s\n', ['Too few subjects (' num2str(x.dataset.nSubjectsSessions) ') to create population-based analysis mask']);
-    x.S.MaskSusceptibility = xASL_im_IM2Column(ones(121,145,121),x.S.masks.WBmask);
-    x.S.VBAmask = x.S.MaskSusceptibility;
-    bSkipStandard = 1;
+    % With too small datasets, created templated are not reliable
+    fprintf('\n\n%s\n\n', ['Only n=' num2str(x.dataset.nSubjectsSessions) ' subject*runs, population-based analysis mask may not be useful']);
+    % x.S.MaskSusceptibility = xASL_im_IM2Column(ones(121,145,121),x.S.masks.WBmask);

```

Comment:
About the title of this entire function. Should we have a wrp_CreateAnalysisMask in ASL module and im_CreateAnalysisMask in population module?

Functionally, this is fine. But it might be slightly confusing to have an IM and WRP functions being disconnected. Perhaps we rename it to im_CreatePopulationMask or something?

Reply:
Yes, good point. See my suggestions above and below after reading your understandable confusion :)